grade_test.go

```diff
@@ -159,7 +162,7 @@ func TestCompile(t *testing.T) {
 	cpp := `#include <iostream>
 	int main() { std::cout << "Hello world!" << std::endl; }`
 
-	tmp, _ := os.CreateTemp("", "*.cpp")
+	tmp, _ := os.CreateTemp(".", "*.cpp")
```

Comment:
Unsure of what this resolves or why this change was needed. Reading the docs on [`os.CreateTemp()`](https://pkg.go.dev/os#CreateTemp),  passing the empty string tells the function to "[use] the default directory for temporary files". Passing `"."` would use the local directory which seems unnecessary to me. If it is is necessary and actually fixes something, let me know and leave a comment. Not blocking since it's a unit test.

Reply:
I think I was just messing around with it to see how it would behave, but I must've forgotten to remove the period. Definitely, going to fix.